src/main/java/org/broadinstitute/consent/http/service/FileStorageObjectService.java

```diff
@@ -0,0 +1,118 @@
+package org.broadinstitute.consent.http.service;
+
+import com.google.cloud.storage.BlobId;
+import org.broadinstitute.consent.http.cloudstore.GCSService;
+import org.broadinstitute.consent.http.db.FileStorageObjectDAO;
+import org.broadinstitute.consent.http.enumeration.FileCategory;
+import org.broadinstitute.consent.http.models.FileStorageObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.ws.rs.NotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+public class FileStorageObjectService {
+
+    GCSService gcsService;
+    FileStorageObjectDAO fileStorageObjectDAO;
+
+    Logger log = LoggerFactory.getLogger(FileStorageObjectService.class);
```

Comment:
See the new-ish `ConsentLogger` for a better way of logging things. We need to be exceptionally careful about logging exceptions as they are sent to Sentry and reported out via slack channels.